Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:alpine3.12
+
+RUN mkdir /nigeriaApi
+
+ADD . /nigeriaApi
+
+WORKDIR /nigeriaApi
+
+COPY . package*.json
```

Comment:
I realized we really don't need this line because we have copied all the files in the root directory into the docker folder.

I hope I am correct @adefemi171